drivers/soundwire/cadence_master.c

```diff
@@ -1352,32 +1352,30 @@ static int cdns_init_clock_ctrl(struct sdw_cdns *cdns)
 	u32 val;
 	u32 ssp_interval;
 	int divider;
-	int freq;
 
 	dev_dbg(cdns->dev, "mclk %d max %d row %d col %d\n",
 		prop->mclk_freq,
 		prop->max_clk_freq,
 		prop->default_row,
 		prop->default_col);
 
+	/* Set frame shape base on the actual bus frequency. */
```

Comment:
this does not set anything as such, the set is left in it's original place..
The comment should remain as well.

Reply:
Exactly, thinks for pointing this out.